src/GraphQL/Types/Composite/ComplexGraphType.cs

```diff
@@ -11,30 +11,50 @@
 
 namespace GraphQL.Types
 {
+    /// <summary>
+    /// Represents a complex (object) graph type.
+    /// </summary>
     public interface IComplexGraphType : IGraphType
     {
+        /// <summary>
+        /// Returns a list of the fields configured for the graph.
+        /// </summary>
         IEnumerable<FieldType> Fields { get; }
 
+        /// <summary>
+        /// Adds a field to the graph.
+        /// </summary>
         FieldType AddField(FieldType fieldType);
 
+        /// <summary>
+        /// Returns true when a field matching the specified name is configured for the graph.
+        /// </summary>
         bool HasField(string name);
 
+        /// <summary>
+        /// Returns the <see cref="FieldType"/> for the field matching the specified name that
+        /// is configured for the graph, or null if none is found.
```

Comment:
```suggestion
        /// is configured for this graph type, or <see langword="null"/> if none is found.
```

I do not insist on using `<see langword="null"/>` instead of `null` and `<see langword="true"/>` instead of `true` (and so on) but if such a notation was provided by the authors of the xml documentation then it probably should be used.